src/reporters/console/console-reporter.js

```diff
@@ -27,9 +27,10 @@ type Row = Array<string>;
 export default class ConsoleReporter extends BaseReporter {
   constructor(opts: Object) {
     super(opts);
-    this._lastCategorySize = 0;
 
+    this._lastCategorySize = 0;
     this.format = (chalk: any);
+    this.isSilent = !!opts.isSilent;
```

Comment:
@bestander Got it. What do you think I should do? The only file that made sense to me during implementation was `console-reporter.js`.